examples/polygon-example/src/app/polygon.nft.mint.express.example.ts

```diff
@@ -0,0 +1,94 @@
+import { TatumPolygonSDK } from '@tatumio/polygon'
+import { Currency, TransactionHash } from '@tatumio/api-client'
+
+const polygonSDK = TatumPolygonSDK({ apiKey: '75ea3138-d0a1-47df-932e-acb3ee807dab' })
+
+export async function polygonNftExpressExample() {
+  // Generate wallet
+  // https://apidoc.tatum.io/tag/Polygon#operation/PolygonGenerateWallet
+  const { mnemonic, xpub } = await polygonSDK.wallet.generateWallet()
+  // Generate private keys
+  // https://apidoc.tatum.io/tag/Polygon#operation/PolygonGenerateAddressPrivateKey
+  const fromPrivateKey = await polygonSDK.wallet.generatePrivateKeyFromMnemonic(mnemonic, 0)
+  const destinationPrivateKey = await polygonSDK.wallet.generatePrivateKeyFromMnemonic(mnemonic, 1)
+  // Generate source and destination addresses
+  // https://apidoc.tatum.io/tag/Polygon#operation/PolygonGenerateAddressPrivateKey
+  const address = polygonSDK.wallet.generateAddressFromXPub(xpub, 0)
+  const to = polygonSDK.wallet.generateAddressFromXPub(xpub, 1)
+
+  // FUND YOUR ACCOUNT WITH MATIC FROM https://faucet.matic.network/
+
+  // Create NFTs on your own smart contract
+  // https://apidoc.tatum.io/tag/NFT-(ERC-721-or-compatible)#operation/NftDeployErc721
+  const nftCreated = (await polygonSDK.nft.deployNFTSmartContract({
+    chain: Currency.MATIC,
+    name: 'HELLO MATIC',
+    symbol: 'HELLO_NFT',
+    fromPrivateKey,
+  })) as TransactionHash
+  console.log(`Created nft with transaction ID: ${nftCreated.txId}`)
+
+  // fetch created contract address from transaction hash
+  // https://apidoc.tatum.io/tag/NFT-(ERC-721-or-compatible)#operation/NftGetContractAddress
+  const { contractAddress } = await polygonSDK.nft.getNFTContractAddress(Currency.MATIC, nftCreated.txId)
+  console.log(`Created NFT smart contract with contract address: ${contractAddress}`)
+
+  // upload your file to the ipfs following this tutorial:
+  // https://docs.tatum.io/guides/blockchain/how-to-store-metadata-to-ipfs-and-include-it-in-an-nft
+
+  // Mint NFT using NTF Express with the pre-built smart contract provided by Tatum
+  // https://apidoc.tatum.io/tag/NFT-(ERC-721-or-compatible)#operation/NftMintErc721
+  const nftMintedExpress = (await polygonSDK.nft.mintNFT({
+    chain: Currency.MATIC,
+    to,
+    // uploaded metadata from ipfs
+    url: 'ipfs://bafybeidi7xixphrxar6humruz4mn6ul7nzmres7j4triakpfabiezll4ti/metadata.json',
+  })) as TransactionHash
+  console.log(`Minted nft with NFT express: transaction ID: ${nftMintedExpress.txId}`)
+
+  // Mint NFT using NTF Express with the pre-built smart contract provided by Tatum
+  // https://apidoc.tatum.io/tag/NFT-(ERC-721-or-compatible)#operation/NftMintErc721
+  const tokenId = 'HELLO-MATIC'
+  // minter address mainnet:
+  const minter = '0x49678AAB11E001eb3cB2cBD9aA96b36DC2461A94'
```

Comment:
add link to https://docs.tatum.io/nft-express/use-nft-express-with-your-own-smart-contract guide